packages/terra-form-textarea/src/terra-dev-site/doc/example/FixedLargeSize.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import Textarea from 'terra-form-textarea/lib/Textarea';
+
+const FixedLargeSizeExample = () => (
+  <Textarea
+    size="large"
+    style={{ resize: 'none' }}
```

Comment:
Why do we need the custom style?

Reply:
I understood "fixed" as non-resizable and the Textarea component did not have a prop for that. May be an misunderstanding. Should I go ahead and remove this?

Reply:
cc/ @neilpfeiffer

Reply:
I think @neilpfeiffer meant fixed as in `isAutoResizable = false`

Reply:
Size full has what you are looking for here.  All sizes except for that one should be resizable.

As for that prop, that dictates if the textarea should be resized automatically as you type in more lines of text instead of hiding the gripper. Should we add another prop that dictates hide gripper? (Note: resize none doesn't work in Internet Explorer)

Reply:
Catching up and refreshing myself with specification details posted https://zpl.io/V09j49E & @emilyrohrbough was correct in my original (poorly worded) intent on the issue:
> I think @neilpfeiffer meant fixed as in `isAutoResizable = false`


\
\
To recap desired state of component functionality:
- Textarea maintains current size prop which is enum one of `small`, `medium`, `large`, `full`
- If size is set to `small`, `medium`, or `large`  -- the gripper always visible, if `isAutoResizable=true` then will grow in height unless user resizes (then stops auto-resize and keeps the user-moved-to size, same as current), and all three sizes are always user-resizable.
- If size is set to `full` -- the gripper is hidden, always grows to width+height of it's container, a user *cannot* resize and *ignores* if `isAutoResizable=true` (since can never grow bigger than it's container).

\
\
To recap & update the desired state for docsite examples:
1. `size="small'` (should be default of `isAutoResizable=false`), 
2. `size="medium"` (should be default of `isAutoResizable=false`), 
3. `size="large"` (should be default of `isAutoResizable=false`), 
4. `isAutoResizable=true` and `size="small"`, 
5.  `size="full"`, 
6. `size="small"` (default `isAutoResizable=false`) that becomes invalid/error when empty, 
7. `size="small"` and `disabled=true`

\
And not captured on the original issue, but I think will help in user understanding of these details, we should update the prop descriptions for isAutoResizable and size: 
```
/**
   * Whether the textarea can be auto-resized vertically. _(Will be ignored if size attribute is set to "full".)_
   */
isAutoResizable: PropTypes.bool,
```

```
/**
   * Standard sizes for a textarea. One of `small`, `medium`, `large`, or `full`.
   * _(Sizes the textarea by setting the rows attribute a corresponding preset value.)_ 
   */
size: PropTypes.oneOf(['small', 'medium', 'large', 'full']),
```

Reply:
Will implement these updates in a separate PR.